config/initializers/error_subscriber.rb

```diff
@@ -38,7 +38,11 @@ def write_log_line(error, handled:, severity:, context:, source: nil)
   def send_to_rollbar(error, handled:, severity:, context:, source: nil)
     context = { handled: handled.present?, source: }.compact.merge(context)
 
-    Rollbar.log(severity_method(severity), error, context)
+    scope = { request: context.delete(:request) }.compact
+
+    Rollbar.scoped(scope) do
```

Comment:
Elevating any `request` context data to be used in `Rollbar.scoped` provides this nice context in the occurrence index view:

![image](https://github.com/user-attachments/assets/07bcb6c9-e357-4620-af63-dc0e64c5e32c)